src/polyswarm_api/core.py

```diff
@@ -539,3 +539,36 @@ def parse_isoformat(date_string):
         return parser.isoparse(date_string)
     else:
         return None
+
+
+def concatenate_api_uri(api, uri):
```

Comment:
This method is very confusing and not really reusable. It should be part of the `ReportTemplate` class at the very least, since it is likely to only ever be used there (probably a protected method). Also, I believe it should probably not exist really, will comment on where it is used.

Reply:
:wastebasket:

Reply:
Removed.